workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -485,29 +489,21 @@ def test_chunks_present(completed_rundir):
     assert completed_rundir.join(CHUNKS_PATH).exists()
 
 
-def test_fv3run_diagnostic_outputs(completed_rundir, configuration):
+def test_fv3run_diagnostic_outputs_check_variables(regtest, completed_rundir):
     """Please do not add more test cases here as this test slows image build time.
     Additional Predictor model types and configurations should be tested against
     the base class in the fv3fit test suite.
     """
-    if configuration == ConfigEnum.nudging:
-        pytest.skip()
-
     diagnostics = xr.open_zarr(str(completed_rundir.join("diags.zarr")))
-    dims = ("time", "tile", "y", "x")
-
-    for variable in [
-        "net_heating",
-        "net_moistening",
-        "physics_precip",
-        "water_vapor_path",
-        "column_integrated_dQu",
-        "column_integrated_dQv",
-    ]:
-        assert diagnostics[variable].dims == dims
+    for variable in sorted(diagnostics):
         assert np.sum(np.isnan(diagnostics[variable].values)) == 0
```

Comment:
Is this still needed if we have checksums now?

Reply:
The checksums don't provide much guidance about what the potential difference is so it's nice to leave this check.

Reply:
OK makes sense